beacon-chain/powchain/service.go

```diff
@@ -297,6 +318,9 @@ func (w *Web3Service) ProcessChainStartLog(depositLog gethTypes.Log) {
 
 // run subscribes to all the services for the ETH1.0 chain.
 func (w *Web3Service) run(done <-chan struct{}) {
+	w.lastHeadInteractTime = time.Now()
+	w.isRunning = true
+	w.runError = nil
```

Comment:
could you initialize `runError` under `NewWeb3Service` when we initalize all the elements of the `Web3Service` struct

Reply:
this line allows to clear previous errors, if run restart